spring-cloud-starter-task-composedtaskrunner/README.adoc

```diff
@@ -152,7 +151,9 @@ $$split-thread-queue-capacity$$:: $$Capacity for Split's  BlockingQueue.
 $$split-thread-wait-for-tasks-to-complete-on-shutdown$$:: $$Whether to wait for scheduled tasks to complete on shutdown, not
  interrupting running tasks and executing all tasks in the queue.
  Default is false;$$ *($$Boolean$$, default: `$$false$$`)*
-
+$$unique-instance-enabled$$:: $$Allows a single ComposedTaskRunner instance to be re-executed. Default
```

Comment:
This name is inaccurate.  It should be something like `increment-instance-enabled` IMHO.  Also, the text should state that it allows a single `ComposedTaskRunner` instance to be re-executed _without changing the parameters_.  You can always re-execute a `ComposedTaskRunner` if you change the parameters.  This allows you to not need to do so.

Reply:
Fixed